### ISSUE TITLE
pipenv fails when inside a directory starting with a dash

### ISSUE THREAD

**Issue by knkski** [kennethreitz/pipenv#439]
The real issue is filed [here](https://github.com/berdario/pew/issues/143), but would it make sense to add `-` to the list of characters to sanitize [here](https://github.com/kennethreitz/pipenv/blob/master/pipenv/project.py#L66)?

A test case:

```
$ git clone http://github.com/foo/bar.git -- "-deleteme"
Cloning into '-deleteme'...
remote: Counting objects: 1176, done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 1176 (delta 121), reused 131 (delta 104), pack-reused 1001
Receiving objects: 100% (1176/1176), 3.81 MiB | 1.42 MiB/s, done.
Resolving deltas: 100% (473/473), done.
$ cd -deleteme/
$ pipenv --three
Creating a virtualenv for this project...
⠋usage: pew [-h] [-p PYTHON] [-i PACKAGES] [-r REQUIREMENTS] [-d] [-a PROJECT]
           envname
pew: error: argument -d/--dont-activate: ignored explicit argument 'eleteme-srGrfIGO'

Virtualenv location: 
```

**Comment by nateprewitt** [comment]
So I don't think there are any technical reasons we can't add `-` to that list, but that's a breaking change and is likely to irritate a lot of users since `-` is a pretty common character.

Before we start making decisions on how to address this though, I just want to make sure I understand the situation. From what I gathered in your issue on the `pew` repository, Jenkins is stripping path names to meet an 80 character max. Is that correct? Have you attempted to use this [suggested solution](https://issues.jenkins-ci.org/browse/JENKINS-38706?focusedCommentId=274531&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-274531) for Jenkins?

**Comment by knkski** [comment]
You are correct on what the issue is, and I did use that solution to successfully fix it for my situation. It's possible that `pipenv` would be used in a situation where it's not possible to use that solution, such as a hosted Jenkins instance, but that is probably too much of an edge case to consider making a breaking change in `pipenv` for, particularly since the real issue lies in `pew`.

**Comment by kennethreitz** [comment]
closing as 'won't fix'
